packages/ppx_monoid/ppx_monoid.0.1/descr

```diff
@@ -0,0 +1,11 @@
+A syntax extension for OCaml to make building values of monoids
+easier. Assumes the existence of two operations in scope for some type
```

Comment:
This file has two fields: the first line and the rest. The first line is a short description that shows up in listings. The rest is a longer description that contains something like you have here. It would be great to have the first line able to stand-alone.